port.js

```diff
@@ -169,52 +169,45 @@ Port.prototype.stop = function stop() {
 };
 
 Port.prototype.request = function request() {
-    var port = this;
-    var $meta = (arguments.length && arguments[arguments.length - 1]) || {};
+    var $meta = arguments.length && arguments[arguments.length - 1];
+    if (!arguments.length) {
+        return Promise.reject(errors.missingParams());
+    } else if (!$meta) {
```

Comment:
Then something like that:
```javascript
if (arguments.length < 1) {
    return Promise.reject(errors.missingParams());
} else if (arguments.length < 2 || !arguments[arguments.length - 1]) {
    return Promise.reject(errors.missingMeta());
}
```
I am not sure what we need to do if arguments[0] is undefined though